packages/data-store/src/types/contact/contact.ts

```diff
@@ -150,9 +150,15 @@ export type Contact = NaturalPerson | Organization
 export type NonPersistedContact = NonPersistedNaturalPerson | NonPersistedOrganization
 export type PartialContact = PartialNaturalPerson | PartialOrganization
 
+export enum PartyOrigin {
+  INTERNAL = 'internal',
```

Comment:
Given it is likely this enum will be used in DTOs have the casing the same!

Reply:
Made all lower case